ts/src/pro/okx.ts

```diff
@@ -1557,27 +1557,20 @@ export default class okx extends okxRest {
         //     { event: 'error', msg: "Illegal request: {"op":"subscribe","args":["spot/ticker:BTC-USDT"]}", code: "60012" }
         //     { event: 'error", msg: "channel:ticker,instId:BTC-USDT doesn"t exist", code: "60018" }
         //
-        const errorCode = this.safeInteger (message, 'code');
+        const errorCode = this.safeString (message, 'code');
         try {
-            if (errorCode) {
+            if (errorCode && errorCode !== '0') {
                 const feedback = this.id + ' ' + this.json (message);
                 this.throwExactlyMatchedException (this.exceptions['exact'], errorCode, feedback);
                 const messageString = this.safeValue (message, 'msg');
                 if (messageString !== undefined) {
                     this.throwBroadlyMatchedException (this.exceptions['broad'], messageString, feedback);
                 }
+                throw new ExchangeError (feedback);
             }
         } catch (e) {
-            if (e instanceof AuthenticationError) {
```

Comment:
Removed this as many times authenticantion errors are thrown from other messageHashes not only 'authenticated'